comp/core/flare/providers.go

```diff
@@ -44,9 +44,9 @@ func (f *flare) collectLogsFiles(fb types.FlareBuilder) error {
 	}
 
 	f.log.Flush()
-	fb.CopyDirToWithoutScrubbing(filepath.Dir(logFile), "logs", shouldIncludeFunc)          //nolint:errcheck
-	fb.CopyDirToWithoutScrubbing(filepath.Dir(jmxLogFile), "logs", shouldIncludeFunc)       //nolint:errcheck
-	fb.CopyDirToWithoutScrubbing(filepath.Dir(dogstatsdLogFile), "logs", shouldIncludeFunc) //nolint:errcheck
+	fb.CopyDirToWithoutScrubbing(filepath.Dir(logFile), "logs", shouldIncludeFunc)                         //nolint:errcheck
+	fb.CopyDirToWithoutScrubbing(filepath.Dir(jmxLogFile), "logs", shouldIncludeFunc)                      //nolint:errcheck
+	fb.CopyDirToWithoutScrubbing(filepath.Dir(dogstatsdLogFile), "logs/dogstatsd_info", shouldIncludeFunc) //nolint:errcheck
```

Comment:
Why the path is changed for dogstatsd logs here?

Reply:
The original code output dogstatsdlogfile into the same path as agent.log whereas it should be inside dogstatsd_info folder instead. So this is just a small fix to a remote config flare's log files placement